app/assets/stylesheets/welcome.scss

```diff
@@ -46,12 +44,99 @@ input#url_original_url {
 
 .btn {
   border-radius: 25px;
+  background: rgba(209, 85, 233, 0.72);
+  border: #9516c7;
+  padding: 6px;
 }
 
 #text_box {
-  margin-top: 50px;
+  margin-top: 5px;
   border-radius: 25px;
   padding: 6px;
-  border: solid 2px rgba(86, 210, 229, 0.72);
+  border: solid 2px #38d5fb;
   outline: none;
+  padding-right: 20px;
+}
+
+li {
+  list-style: none;
+  color: #38d5fb;
+  font-size: 15px;
+}
+
+a:hover {
+  color: #38d5fb;
+  text-decoration: none;
+}
+
+a {
+  color: white;
+}
+
+.font-icon {
+  color: #ff925c;
+}
+
+// .create-shortened-url {
+//   padding: 50px;
+// }
+
+.url-list {
+  background: black;
+}
+
+#url-title {
+  margin-top: 20px;
+}
+
+/* CSS used here will be applied after bootstrap.css */
+.mini-submenu{
+  display:none;
+  background-color: rgba(0, 0, 0, 0);
+  border: 1px solid rgba(0, 0, 0, 0.9);
+  border-radius: 4px;
+  padding: 9px;
+  /*position: relative;*/
+  width: 42px;
+
+}
+
+.mini-submenu:hover{
+  cursor: pointer;
+}
+
+.mini-submenu .icon-bar {
+  border-radius: 1px;
+  display: block;
+  height: 2px;
+  width: 22px;
+  margin-top: 3px;
+}
+
+.mini-submenu .icon-bar {
+  background-color: #000;
+}
+
+.filter-text {
+  margin-right: 500px;
+}
+
+#slide-submenu{
+  background: rgba(0, 0, 0, 0.45);
+  display: inline-block;
+  padding: 0 8px;
+  border-radius: 4px;
+  cursor: pointer;
+}
+
+.search-bubble {
+  background: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.